ArchiSteamFarm/Commands.cs

```diff
@@ -689,6 +691,44 @@ internal sealed class Commands {
 			return responses.Count > 0 ? string.Join(Environment.NewLine, responses) : null;
 		}
 
+		private string ResponseBalance(ulong steamID) {
+			if (steamID == 0) {
+				Bot.ArchiLogger.LogNullError(nameof(steamID));
+				return null;
+			}
+
+			if (!Bot.IsMaster(steamID)) {
+				return null;
+			}
+
+			if (!Bot.IsConnectedAndLoggedOn) {
```

Comment:
Bot doesn't need to be connected for this.

Reply:
Although on the second look it's good to have it in place so leave it be, people won't get false impression or outdated balances this way.